Asset loading from v1 - #861m70z4x

OswaldUnity opened the pull request:
- Fix loading v1 or new scenes since no scene existed before
- Fix getting right guid from gltf assets
  - Was looking for the wrong guid

OswaldUnity committed:
Fix loading v1 scenes since no scene existed b4

OswaldUnity committed:
Fix getting right guid from gltf assets

OswaldUnity committed:
Update rename new function

MGH-Automator commented:
Task linked: [CU-861m70z4x Load from version 1 asset loading is not working](https://app.clickup.com/t/861m70z4x)

cblech reviewed (changes requested):
This does not seem to work with assets, that are not originally from the builder. Consider [this ](https://github.com/metagamehub/dcl-edit-testing/tree/main/projects/alpha-with-assets) project. Fireplace, Sign, and Tree work, but the Popcornmaschine just shows an error

OswaldUnity committed:
Added ClassModels folder
Removed old v1 asset loading on v2
Moved AssetsJsonWrapper to ClassModels

OswaldUnity committed:
Added GlobalConstants file
This is needed in order to avoid changing multiple places instead of just one

OswaldUnity committed:
Added V1AssetLoaderSystem
Handles assets from v1 projects, assets from asset.json file
Added FileIOUtility - Handles errors on file io functions

OswaldUnity committed:
Added missing meta files

OswaldUnity committed:
Added write object as json string to file

cblech commented:
Will be solved differently
